data/ecosystem/vendors.yaml

```diff
@@ -132,6 +132,13 @@
   contact: 'support@hyperdx.io'
   oss: true
   commercial: true
+- name: Immersive Fusion
+  distribution: true
```

Comment:
can you provide a link where we can understand/see the distribution you have?

Reply:
That was oversight. I've submitted a fix. Thank you for catching.